Library/Formula/opentsdb.rb

```diff
@@ -0,0 +1,104 @@
+require "socket"
+class Opentsdb < Formula
+  desc "Scalable, distributed Time Series Database."
+  homepage "http://opentsdb.net/"
+  url "https://github.com/OpenTSDB/opentsdb/releases/download/v2.2.0/opentsdb-2.2.0.tar.gz"
+  sha256 "5689d4d83ee21f1ce5892d064d6738bfa9fdef99f106f45d5c38eefb9476dfb5"
+
+  option "without-lzo", "Don't use LZO Compression"
+
+  if build.without? "lzo"
+    depends_on "hbase"
+  else
+    depends_on "hbase" => "with-lzo"
+  end
+  depends_on :java => "1.6+"
+  depends_on "gnuplot" => :optional
+
+  def install
+    # submitted to upstream https://github.com/OpenTSDB/opentsdb/pull/711
+    # mkdir_p is called from in a subdir of build so needs an extra ../ and there is no rule to create $(classes) and
+    # everything builds without specifying them as dependencies of the jar.
+    inreplace "Makefile.in" do |s|
+      s.sub!(/(\$\(jar\): manifest \.javac-stamp) \$\(classes\)/, '\1')
+      s.sub!(/(echo " \$\(mkdir_p\) '\$\$dstdir'"; )/, '\1../')
+    end
+
+    mkdir "build" do
+      system "../configure",
+             "--disable-silent-rules",
+             "--prefix=#{prefix}",
+             "--mandir=#{man}"
+      system "make"
+      system "make", "install-exec-am"
+      system "make", "install-data-am"
+    end
+
+    (var/"cache/opentsdb").mkpath
+  end
+
+  def post_install
+    etc.install Dir["#{opt_share}/opentsdb/etc/opentsdb"]
+
+    system "#{Formula["hbase"].opt_libexec}/bin/start-hbase.sh"
+
+    ENV["HBASE_HOME"]=Formula["hbase"].opt_libexec.to_s
```

Comment:
Please leave an empty space rather than joining these, it makes it quite a bit easier to understand at quick glance:
```ruby
 ENV["HBASE_HOME"] = Formula["hbase"].opt_libexec
```
etc.